src/pal/src/configure.cmake

```diff
@@ -933,6 +933,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
   set(PAL_PT_READ_D PT_READ_D)
   set(PAL_PT_WRITE_D PT_WRITE_D)
   set(HAS_FTRUNCATE_LENGTH_ISSUE 1)
+  set(HAVE_SCHED_OTHER_ASSIGNABLE 1)
```

Comment:
Ah, you mean like:

```
p = default_val

if (some_condition)
   p = special_val
```

pattern is not readable and we should always prefer

```
if (some_condition)
  p = special_val
else
  p = default_val
```

instead? I wonder does it also hold in case of Cmake where it predefines some default flag values and users overwrite those based on the project requirement.